buildSrc/src/main/kotlin/net/skyscanner/backpack/tokens/BpkColors.kt

```diff
@@ -58,16 +65,10 @@ object BpkColor {
       parseColors(source, resolveReferences = true) { it.isSemanticColor() && !it.isPrivate }.toBpkColors()
   }
 
-  object Internal : BpkParser<Map<String, Any>, Map<String, BpkColors>> {
+  object Internal : BpkParser<Map<String, Any>, BpkColors> {
```

Comment:
moved the mapping to the formatter to keep the xml version simple/in one file

Reply:
We may be needed to re-package it in the future

Reply:
yep easy enough to do if we need to - it's better now than it was before - at least now the private ones are in a separate file